src/ddmd/statement.d

```diff
@@ -107,6 +124,7 @@ TypeIdentifier getThrowable()
 
 
 /***********************************************************
+ * Specification: http://dlang.org/spec/statement.html
```

Comment:
I proposed this a while ago:

https://github.com/dlang/dmd/pull/6877
https://github.com/dlang/dlang.org/pull/1683

Tl;Dr: @CyberShadow was worried about the added complexity to the dlang.org build process